locales/en/email-strings.ftl

```diff
@@ -160,3 +160,19 @@ faq-v2-3 = I just found out I’m in a data breach. What do I do next?
 
 # Link Title
 faq-v2-4 = How does {-product-name} treat sensitive sites?
+
+
+# This is the title of a link that is nested inside { pre-fxa-subhead }.
```

Comment:
This approach is painful but usable when it comes to a short link (like sign-in), patching together entire sentences is less than ideal, and should be avoided if possible.

How much code could we have in a string? Could we use a span, for example, and then replace it with a link? That would make things less confusing.

Reply:
@flodolo We could, but I think that would introduce its own set of issues and edgecases. @betsymi Are you ok with only reducing the clickable/linked portion of this string to "Firefox Account" ?

Reply:
I'm OK with that, but account should be lowercased. We are moving away from capital A Account.

Reply:
@betsymi thanks! In that case, do we think "account" should now be localized? And should this change be reflected everywhere we see "Firefox Account"?

Reply:
> @flodolo We could, but I think that would introduce its own set of issues and edgecases. @betsymi Are you ok with only reducing the clickable/linked portion of this string to "Firefox Account" ?

Can you clarify? What's I'm asking is to have something like

```
create-a-free-fxa = <a>Create a free {-brand-fxa}</a>, and you can add up to 15 email addresses.
```

That shouldn't affect how much of the sentence you linkify.